wiki/Storyboard/Scripting/General_Rules/en.md

```diff
@@ -2,103 +2,109 @@
 
 ## Basic Rules
 
 ### Objects
 
-- An "[object](/wiki/Storyboard/Scripting/Objects)" in the following sections is an instance of a sprite or an animation in your storyboard. Storyboards can also have sound; see [that section](/wiki/Storyboard/Scripting/Audio) for more details.
-- The accepted formats are PNG and JPEG.
-  - JPEG is lossy, which means it's smaller in filesize, but isn't pixel-for-pixel exactly what was input. It also doesn't support transparency. Therefore, it's good for backgrounds and for square, photo-realistic images.
-  - PNG is lossless, which means it retains pixel-for-pixel information, but is of a larger filesize than JPEG. It supports transparency, so it's usually the best for foreground objects / text.
-- Animation is done in-engine; don't use PNG's layers or animation features. Instead, save each frame as a separate file and name the files with a decimal number before the extension (e.g., "sample0.png", "sample1.png" for a 2-frame animation "sample.png").
+*For objects in [osu!](/wiki/Game_mode/osu!) and [Beatmapping](/wiki/Beatmapping), see: [Hit Objects](/wiki/Gameplay/Hit_object)*
+
+A [Storyboard object](/wiki/Storyboard/Scripting/Objects) is an instance of a sprite or an animation in a storyboard. Storyboards can also have sound, see the [Audio](/wiki/Storyboard/Scripting/Audio) guide for more details.
+
+The accepted formats for objects are PNG and JPEG. JPEG is lossy, which means it is smaller in file size, but does not precisely save each pixel. It also does not support transparency. Therefore, it is good for backgrounds and for square or photo-realistic images. PNG is lossless, which means it retains pixel-by-pixel information, but is of a larger file size than JPEG. It supports transparency, so it is usually the best for foreground objects / text.
+
+Animations are done in-engine, so the PNG layer system or animation features should not be used. Instead, save each frame as a separate file and name the files with a decimal number before the extension (e.g., "sample0.png", "sample1.png" for a 2-frame animation "sample.png").
 
 ### Screen size
 
-![Editor screen size. Green is screen size and Red is playarea](img/SBS_SS.jpg "Editor screen size. Green is screen size and Red is playarea")
+![Editor screen size. Green is screen size and Red is play area](img/SBS_SS.jpg "Editor screen size. Green is screen size and Red is play area")
+
+The editor screen is 640 x 480 pixels and the general play area is 510 x 385 pixels.
+
+Coordinates are specified with positive values for `X` going to the **right**, positive values for `Y` going **down**, and the origin (0,0) being placed at the upper-left corner of the screen. This is different from traditional [Euclidean coordinate systems](https://en.wikipedia.org/wiki/Euclidean_space), but is the same as most computer-graphics systems. It is possible to specify coordinates to be outside of these boundaries (e.g., for a sprite to come in from off-screen).
```

Comment:
Not sure if it's worth to mention euclidean space here at all.

Reply:
probably not, but it's a task for another time